website_multi_theme/README.rst

```diff
@@ -0,0 +1,149 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===================
+Website Multi Theme
+===================
+
+Allow the website admin to set a different theme for each website.
+
+Installation
+============
+
+To make this module work, you need to either:
+
+* Install any of the officially supported themes:
+
+  * theme_bootswatch
+
+* Install any of the unofficially supported themes (at your own risk):
+
+  * theme_anelusia
+  * theme_artists
+  * theme_avantgarde
+  * theme_beauty
+  * theme_bewise
+  * theme_bistro
+  * theme_bookstore
+  * theme_clean
+  * theme_enark
+  * theme_graphene
+  * theme_kea
+  * theme_loftspace
+  * theme_mongolia
+  * theme_nano
+  * theme_notes
+  * theme_odoo_experts
+  * theme_orchid
+  * theme_treehouse
+  * theme_vehicle
+  * theme_yes
+  * theme_zap
+
+Themes in the above lists will become multiwebsite when installed along this
+module. **If they get installed after ``website_multi_theme``, update this
+module manually**, or it will not be notified of such change.
```

Comment:
I don't understand what is wrong here and why do you call it "patch". 
``_multi_theme_update`` doesn't make monkey patch or something like that. It's a function that author asks user to call it manually (via updating the module), while I suggest to do that automatically by moving it to ``init``